packages/quantic/force-app/main/default/lwc/quanticNoResults/quanticNoResults.html

```diff
@@ -0,0 +1,20 @@
+<template>
+    <template if:false={hasResults}>
+        <div class="slds-grid slds-wrap">
```

Comment:
Using [`slds-grid_vertical`](https://www.lightningdesignsystem.com/utilities/grid/#Top-to-Bottom) would make the layout a tad simpler.